src/book.py

```diff
@@ -82,7 +82,37 @@ def __init__(self, filepath=None, rawtext=None, encoding='utf-8', file_format='s
                 else:
                     self._title = None
         else:
-            self._title = None
+            self._title = title #FIXME IN MASTER
+            
+    def __add__(self, other): 
+        '''
+        Overloaded the addition operator for BookText. Returns text of both the booktext objects, and carries over
+        the author and title info. If authors or titles are not same, the 1st book is preferred over the second.
+        '''
+        author1 = self._author
+        author2 = other._author
+        title1 = self._title
+        title2 = other._title
+        if (author1 is None and author2 is not None):
+            author = author2
+        elif (author1 is not None and author2 is None):
+            author = author1
+        elif (author1 != author2):
+            print ("The authors are not the same. Assigning the author of the 1st Book to the resultant bookobject")
+            author = author1
+        else:
+            author = author1
+            
+        if (title1 is None and title2 is not None):
+            title = title2
+        elif (title1 is not None and title2 is None):
+            title = title1
+        elif (title1 != title2):
+            print ("The authors are not the same. Assigning the title of the 1st Book to the resultant bookobject")
+            title = title1
+        else:
+            title = title1
+        return BookText(rawtext=self._text+other._text, author=author, title=title, meta=None) 
```

Comment:
```suggestion
        return BookText(rawtext=self._text+' '+other._text, author=author, title=title, meta=None) 
```
Let's add a space between subsequent texts for readability.

Reply:
Ok. This is better.